tests/tck/features/yield/parameter.feature

```diff
@@ -7,16 +7,48 @@ Feature: Parameter
     Given a graph with space named "nba"
     Given parameters: {"p1":1,"p2":true,"p3":"Tim Duncan","p4":3.3,"p5":[1,true,3],"p6":{"a":3,"b":false,"c":"Tim Duncan"},"p7":{"a":{"b":{"c":"Tim Duncan","d":[1,2,3,true,"Tim Duncan"]}}},"p8":"Manu Ginobili", "p9":["Tim Duncan","Tony Parker"]}
 
-  Scenario: return parameters
+  Scenario: [param-test-001] without define param
+    When executing query:
+      """
+      RETURN $p_not_exist AS v
+      """
+    Then a SyntaxError should be raised at runtime: Direct output of variable is prohibited near `$p_not_exist'
+
+  Scenario: [param-test-002] support null
+    When executing query:
+      """
+      RETURN $p1 is not null AS v
```

Comment:
Need to add some more complex expression scenarios, such as expression nesting, type checking, runtime error, etc.

Reply:
done